tools/webpack.config.ts

```diff
@@ -250,7 +250,6 @@ export default (env?: string): webpack.Configuration[] => {
         {path: "../static/third/marked/lib/marked.js"},
         {path: "../static/js/debug.js"},
         {path: "jquery/dist/jquery.js", name: ["$", "jQuery"]},
-        {path: "underscore/underscore.js", name: "_"},
         {path: "handlebars/dist/cjs/handlebars.runtime.js", name: "Handlebars"},
         {path: "sortablejs/Sortable.js"},
         {path: "winchan/winchan.js", name: "WinChan"},
```

Comment:
What's the effect of this hunk?

Reply:
This line previously installed a global `_` as a side effect when `underscore/underscore.js` was imported.

Reply:
I meant more about whether it has an effect on the debugging issues noted in the comment above.

Reply:
No. The comment above is a non-sequitur. We should be using the unminified versions of everything, because we minify all our bundles ourselves with source maps. `underscore/underscore.js` is unminified; it should have been simply `underscore` which is unminified; we’re replacing it with `lodash` which is unminified.

Reply:
OK; maybe worth deleting that comment then.